src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,28 +1,41 @@
 package edu.kis.vh.nursery;
 
+import edu.kis.vh.nursery.storage.IntArrayStack;
+import edu.kis.vh.nursery.storage.IntStorageInterface;
+
 public class FIFORhymer extends DefaultCountingOutRhymer {
+	
+	private final IntArrayStack TEMP = new IntArrayStack();

```

Comment:
This object is not a constant (it is mutable), the name should be written lowercase.
Wrong choice of class (correctness more important than efficiency)